chapters/classes.tex

```diff
@@ -1431,6 +1431,8 @@ \subsection{Real Type}\label{real-type}
 The \lstinline!nominal! attribute is meant to be used for scaling purposes and to
 define tolerances in relative terms, see \cref{attributes-start-fixed-nominal-and-unbounded}.
 
+As a fall-back if there is neither value nor start-value then the closest value to \lstinline!0.0! consistent with the bounds (as default \lstinline!0.0!) shall be used.
```

Comment:
There seems to be a missing context for "fall-back" here.  Fall-back for what?  Perhaps something along the lines of this:
```suggestion
When the \lstinline!start!-attribute is not provided, the fall-back value for use in algorithm initialization etc is the closest value to \lstinline!0.0! consistent with the bounds (as default \lstinline!0.0!) shall be used.
```

Then, all places relying on this fall-back should also be updated, and the "etc" part should probably be expanded.

Reply:
Sorry, I understand this sentence if I ignore the part in the brackets: "(as default \lstinline!0.0!)" But I am not sure what "as default \lstinline!0.0!" is trying to say.

Reply:
Isn't it just an attempt at pointing out that the number closest to 0.0 will be 0.0 if there are no constraints (which is the default).

Reply:
It's attempting to state the rather obvious fact that if min and max aren't specified the closest value to 0.0 consistent with min&max is 0.0 (!). It might be simpler to just remove it.

Reply:
My preference would be to remove it.

Reply:
I note that the way we write "etc" is far from consistent, with lots of both "etc" and "etc.".  I think it should be with a period, and it seems we need a note in the style guide considering the current inconsistency (in addition to actually changing all cases to the proper form).

Combined with @eshmoylova's suggestion, this gives:
```suggestion
When the \lstinline!start!-attribute is not provided, the fallback value for use in algorithm initialization etc., is the closest value to \lstinline!0.0! consistent with the bounds shall be used.
```

Still, it would be good to expand the "algorithm initialization etc." part.  If nothing else, that should help us see the scope of this PR.

Reply:
That part has now been removed, and fall-back value defined more clearly (once).